openlibrary/templates/trending.html

```diff
@@ -30,5 +30,7 @@ <h1>$_('Trending Books'): $(pages[mode])</h1>
   </div>
   <div class="pager">
     $:macros.Pager(page, 200, results_per_page=20)
+    $if mode == 'forever':
+    <p class="collection-date">$_("Data Collection Started: October 2017")</p>
```

Comment:
```suggestion
    $if mode == 'forever':
      <p>$_("Earliest trending data is from October 2017")</p>
```
Be sure to indent code in `if` blocks an appropriate amount.
I think that this can be moved to the top of the page, just above the `ul.nav-bar` element, like so:

![image](https://github.com/internetarchive/openlibrary/assets/28732543/45f9691f-1dbd-4b3a-b2b9-9aaaed390728)